### PR TITLE
Add source links to the open-telemetry documentation

### PR DESCRIPTION
**Pull request by mziolekda**

### CODE REVIEW

File: docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst [hunk 1]
```diff
@@ -51,31 +51,12 @@ Add Project Dependencies
...
-      <artifactId>opentelemetry-grpc-1.6</artifactId>
-      <version>x.y.z</version>
-    </dependency>
+.. remoteliteralinclude:: https://raw.githubusercontent.com/digital-asset/ex-java-bindings-with-opentelemetry/master/pom.xml
+   :language: xml
+   :lines: 33-52
 
 .. note::
-    Replace ``x.y.z`` for both dependencies with the version that you want to use. You can find the available versions by checking the `Maven Central Repository <https://search.maven.org/artifact/io.opentelemetry/opentelemetry-api>`__ etc.
+    Replace the version number in all dependencies with the version that you want to use. You can find the available versions by checking the `Maven Central Repository <https://search.maven.org/artifact/io.opentelemetry/opentelemetry-api>`__ etc.
```
Comment (marciw-da) [inline docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst:59]:
```suggestion
    Replace the version number in each dependency with the version you want to use. To find available versions, check the `Maven Central Repository <https://search.maven.org/artifact/io.opentelemetry/opentelemetry-api>`__.
```

Comment (marciw-da) [reply docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst:59]: The Maven search doesn't show anything for me... maybe double-check the link

**Review by marciw-da — COMMENTED** [review]
Small suggestion

File: docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst (hunk 1 above)
Comment (mziolekda) [reply docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst:59]: It gets me to this page:
image

Comment (mziolekda) [reply docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst]: I have also asked a few colleagues to check, and it works for them too, :thinking:

Comment (mziolekda) [reply docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst]: It also works for me when I click it from the current documentation page (little note at the bottom):
https://docs.daml.com/2.8.0/app-dev/bindings-java/open-tracing.html#add-project-dependencies

Comment (marciw-da) [reply docs/2.8.0/docs/app-dev/bindings-java/open-tracing.rst]: Thanks, the link works fine for me now. I must have hit a Maven hiccup or outage yesterday.

**Review by marciw-da — APPROVED** [review]
LGTM, thanks!

**Pull request merged**